composer.json

```diff
@@ -14,14 +14,14 @@
     }],
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "^2.7 || ^3.0",
-        "symfony/yaml": "^2.7 || ^3.0"
+        "symfony/framework-bundle": "^2.7 || ^3.0 || ^4.0",
+        "symfony/yaml": "^2.7 || ^3.0 || ^4.0"
     },
     "require-dev": {
         "doctrine/cache": "1.*",
         "predis/predis": "^1.0",
-        "symfony/console": "^2.7 || ^3.0",
-        "symfony/phpunit-bridge": "^2.7 || ^3.0",
+        "symfony/console": "^2.7 || ^3.0 || ^4.0",
+        "symfony/phpunit-bridge": "^2.7 || ^3.0 || ^4.0",
         "phpunit/phpunit": "4.8.*"
```

Comment:
the version constraint here should now be `^4.8.35 || ^5.4.3 || ^6.0`

Reply:
Yeah, good idea!